index.html

```diff
@@ -1,13 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <title></title><link rel="stylesheet" href="style.css">
+    <title>Salmon Cookies</title><link rel="stylesheet" href="mainstyle.css">
+    <link href="https://fonts.googleapis.com/css?family=Pangolin" rel="stylesheet">
   </head>
   <body>
-    <section id="main-content">
-      <h1>Cookie Stand</h1>
-    </section>
-    <script src="app.js" type="text/javascript"></script>
-    </body>
+    <h1>Welcome to the Salmon Cookies Website</h1>
+    <img src="https://raw.githubusercontent.com/codefellows/seattle-201d19/c22e563a40184ee90e8a1cfb930f6aa7e65b3571/class-06-dom-objects-domain_modeling/assets/salmon.png">
```

Comment:
It'll be easier if you download the image and save it locally inside of your repo, then use a local filepath instead.